miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -61,29 +61,20 @@ const.FAUCET_ACCOUNT=536870912 # 0b00100000_00000000_00000000_00000000
 # Specifies a minimum number of ones for a valid account ID.
 const.MIN_ACCOUNT_ONES=5
 
-# The depth of the account storage Sparse Merkle Tree
-const.STORAGE_TREE_DEPTH=8
-
-# The depth of the account code tree
-const.ACCOUNT_CODE_TREE_DEPTH=8
-
 # The account storage slot at which faucet data is stored.
-# Fungible faucet: The faucet data consists of [EMPTY_WORD, EMPTY_WORD, EMPTY_WORD, total_issuance]
+# Fungible faucet: The faucet data consists of [0, 0, 0, total_issuance]
 # Non-fungible faucet: The faucet data consists of SMT root containing minted non-fungible assets.
-const.FAUCET_STORAGE_DATA_SLOT=254
-
-# The account storage slot at which the slot types commitment is stored.
-const.SLOT_TYPES_COMMITMENT_STORAGE_SLOT=255
+const.FAUCET_STORAGE_DATA_SLOT=0
 
 # The maximum storage slot index
 const.MAX_STORAGE_SLOT_INDEX=255
 
-# The maximum value a slot type can take (An array of depth 64).
-const.MAX_SLOT_TYPE=64
-
 # The maximum number of account interface procedures.
 const.MAX_NUM_PROCEDURES=256
 
+# The maximum number of account storage slots.
+const.MAX_NUM_STORAGE_SLOTS=256
```

Comment:
Related to a previous comment: let's make this 255.

`MAX_STORAGE_SLOT_INDEX` above would need to be updated accordingly. In fact, we can probably define one constant in terms of another.

Reply:
I did not find an example of a `const` defined in MASM in terms of another. 

```
# The maximum storage slot index
const.MAX_STORAGE_SLOT_INDEX=254

# The maximum number of account storage slots.
const.MAX_NUM_STORAGE_SLOTS=MAX_STORAGE_SLOT_INDEX+1
```

After testing it seems to work correctly.

Reply:
The ability to have computed constants is relatively new but it is covered [here](https://0xpolygonmiden.github.io/miden-vm/user_docs/assembly/code_organization.html#constants).